Description: Include definition of Prem function from factory.
 Upstream patches factory directly to make this public.  See [1].
 .
 [1] https://github.com/Macaulay2/M2/blob/master/M2/libraries/factory/patch-4.0.0%2Bm4
Author: Doug Torrance
Last-Update: 2015-06-04

--- a/M2/Macaulay2/e/x-factor.cpp
+++ b/M2/Macaulay2/e/x-factor.cpp
@@ -10,6 +10,14 @@
 
 #define Matrix FactoryMatrix
 #include <factory/factory.h>             // from Messollen's libfac
+CanonicalForm
+Prem (const CanonicalForm& F, const CanonicalForm& G);
+
+CanonicalForm
+Premb (const CanonicalForm &f, const CFList &L);
+
+CanonicalForm
+Prem (const CanonicalForm &f, const CFList &L);
 #undef INT64
 #undef Matrix
 #undef ASSERT
